Misc/NEWS.d/next/Library/2018-05-08-15-01-10.bpo-33365.SicsAd.rst

```diff
@@ -0,0 +1 @@
+print the header values besides the keys (debuglevel=1)
```

Comment:
It looks unclear out of the context. Please explain what is changed.

Add a period, start the sentence with a title case, and please add "Path by *yourname*."